micrometer-tracing-bridges/micrometer-tracing-bridge-brave/src/main/java/io/micrometer/tracing/brave/bridge/BraveFinishedSpan.java

```diff
@@ -70,6 +70,8 @@ public FinishedSpan setName(String name) {
     }
 
     @Override
+    // TODO figure out what to do about name nullability
+    @SuppressWarnings("NullAway")
     public String getName() {
         return this.mutableSpan.name();
     }
```

Comment:
This is a TODO for figuring out after this PR. Brave API marks the name nullable, but FinishedSpan API does not.